Test_design_logging .md

```diff
@@ -0,0 +1,33 @@
+# Test design of the "Login with email address" functionality of the pracuj.pl website
+
+> ### Login with email address
+
+1. Checking the visibility of all elements of the login form 
+
+* [Display all fields, buttons and links](#general-information)
+
+2. Verification of login with correct data 

```

Comment:
Valid/invalid or correct/incorrect chose one in point 2 and 3 ;)

Reply:
You mean the consistency of vocabulary in both points? If so it would be like this: login with invalid/valid data